src/Microsoft.Azure.WebJobs/TableAttribute.cs

```diff
@@ -91,7 +91,7 @@ public string RowKey
         /// <summary>
         /// Allow arbitrary table filter. RowKey should be null. 
         /// </summary>
-        [AutoResolve]
+        [AutoResolve(ResolutionPolicyType = typeof(ODataFilterResolutionPolicyMarker<TableAttribute>))]
```

Comment:
><TableAttribute> [](start = 84, length = 16)

This generic here feels completely out of place. It has no semantic value here; and is only here because it leaked up from the implementation (and the implementation only needs it because of a rare SQL case that no other binding is using)